### PR TITLE
Introduce IntegrityError

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

**Review by YoonhoKim0527 — APPROVED** [review]

File: cli/src/main.rs [hunk 1]
```diff
@@ -69,7 +49,36 @@ async fn main() -> eyre::Result<()> {
...
+}
+
+#[tokio::main(flavor = "multi_thread")]
+#[allow(unreachable_code)]
+async fn main() -> eyre::Result<()> {
+    color_eyre::install().unwrap();
+    env_logger::init();
+
+    let private_key = std::env::args().nth(1).unwrap();
+    let sorc = std::env::args().nth(2).unwrap();
```
Comment (TomTaehoonKim) [inline cli/src/main.rs:62]:
```suggestion
    let server_or_client = std::env::args().nth(2).unwrap();
```

File: cli/src/main.rs [hunk 2]
```diff
@@ -17,27 +17,7 @@ fn to_commit_hash(s: &str) -> Result<CommitHash> {
...
-    }
-
-    return Ok(());
-
-    let args = cli::Cli::parse();
-    let path = args.path.display().to_string();
-    let config: Config =
-        serde_spb::from_str(&tokio::fs::read_to_string(&format!("{}/config.json", path)).await?)?;
-
+async fn run(args: cli::Cli, path: String, config: Config) -> eyre::Result<()> {
```
Comment (TomTaehoonKim) [inline cli/src/main.rs:20]: What is the path for?

Comment (junha1) [reply cli/src/main.rs:20]: see the line 28.

Comment (TomTaehoonKim) [reply cli/src/main.rs:20]: So, it's a path to the repository?

Comment (junha1) [reply cli/src/main.rs:20]: path of the root directory of a Simperby blockchain.  See `directory.md`

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Pull request merged**